website/layouts/language.erb

```diff
@@ -0,0 +1,1030 @@
+                        <a href="/docs/provisioners/chef.html">chef</a>
+                      </li>
+
+                      <li>
+                        <a href="/docs/provisioners/habitat.html">habitat</a>
+                      </li>
+
+                      <li>
+                        <a href="/docs/provisioners/puppet.html">puppet</a>
+                      </li>
+
+                      <li>
+                        <a href="/docs/provisioners/salt-masterless.html">salt-masterless</a>
+                      </li>
+                    </ul>
+                  </li>
+
+                </ul>
+              </li>
+
+            </ul>
+          </li> <!-- resources -->
+
+          <li>
+            <a href="#">Providers</a>
+            <ul class="nav">
+              <li>
+                <a href="/docs/configuration/blocks/providers/index.html">Overview</a>
+              </li>
+
+              <li>
+                <a href="/docs/configuration/provider-requirements.html">Provider Requirements</a>
+              </li>
+
+              <li>
+                <a href="/docs/configuration/providers.html">Provider Configuration</a>
+              </li>
+
+              <li>
+                <a href="/docs/configuration/dependency-lock.html">Dependency Lock File</a>
+              </li>
+            </ul>
+          </li>
+
+          <li>
+            <a href="/docs/configuration/variables.html">Input Variables</a>
+          </li>
+
+          <li>
+            <a href="/docs/configuration/outputs.html">Output Values</a>
+          </li>
+
+          <li>
+            <a href="/docs/configuration/locals.html">Local Values</a>
+          </li>
+
+          <li>
+            <a href="#">Module Calls</a>
+            <ul class="nav">
+              <li>
+                <a href="/docs/configuration/blocks/modules/index.html">Overview</a>
+              </li>
+
+              <li>
+                <a href="/docs/configuration/modules.html">Module Blocks</a>
+              </li>
+
+              <li>
+                <a href="/docs/modules/sources.html">Module Sources</a>
+              </li>
+
+            </ul>
+          </li><!-- module calls -->
+
+          <li>
+            <a href="/docs/configuration/data-sources.html">Data Sources</a>
+          </li>
+
+          <li>
+            <a href="#">Backends</a>
+            <ul class="nav">
+              <li>
+                <a href="/docs/configuration/blocks/backends/index.html">Overview</a>
+              </li>
+
+              <li>
+                <a href="/docs/configuration/backend.html">Backend Configuration</a>
+              </li>
+
+              <li>
+                <a href="#">Enhanced Backends</a>
+                <ul class="nav nav-auto-expand">
+                  <li>
+                    <a href="/docs/backends/types/local.html">local</a>
+                  </li>
+                  <li>
+                    <a href="/docs/backends/types/remote.html">remote</a>
```

Comment:
Strongly doubt it. (_Theoretically_ anyone could build something TFC-like that obeys the protocol, but practically we designed that protocol around our own commercial needs and change it at our own convenience.) 

The name of that backend is kind of a thorn in my side re: explainability, yeah. 😑 I don't know what to do about that either. Could put like `remote (TFC/TFE)` or something, except I expect we all hate that.

Reply:
There is already at least one third-party implementation of the server side of the remote backend protocol, but I would agree that it's debatable that we would be talking about this as an intentional extension point any time soon, because (unlike our other remote service protocols) this thing is hooked up directly to the main Terraform Cloud/Enterprise API, not something generic that is documented as a third-party extension point.

Reply:
Haha. You're right Nick I hate it 😛 Let's leave it for now